presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -283,4 +285,17 @@ public QueryManagerConfig setQueryExecutionPolicy(String queryExecutionPolicy)
         this.queryExecutionPolicy = queryExecutionPolicy;
         return this;
     }
+
+    @Min(1)
+    public int getMinWorkerNodes()
+    {
+        return minWorkerNodes;
+    }
+
+    @Config("query.min-worker-nodes")
```

Comment:
This config name is pretty confusing - it seems like this is the min worker nodes required for every query (i.e. the min distribution required for the query). It also indicates that queries will not run unless this number is met (which is not true - this is only initial)

My idea would be to call this `query-manager.min-worker-nodes-initialized`.